src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -23,6 +23,9 @@ including:
 - Local maximum/minimum Zalesak-type limiting for conservative variables (`local_minmax_variables_cons`)
 - Positivity limiting for conservative variables (`positivity_variables_cons`)
 
+Conservative variables to be limited are passed as a strings, e.g. `local_minmax_variables_cons = ["rho"]`
```

Comment:
You should also fix the defaults in the docstring (above) and the actual ctor (below) to
```julia
                      local_minmax_variables_cons = String[],
                      positivity_variables_cons = String[],
```

Reply:
Yes thanks :+1: